wazo_amid_client/exceptions.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, response):
         except (TypeError, KeyError):
             raise InvalidAmidError()
 
-        super().__init__(f'{self.message}', response=response)
+        super(HTTPError, self).__init__(f'{self.message}', response=response)
```

Comment:
Not sure why we are inheriting from `AmidError` if we are skipping it's constructor entirely.  Wouldn't it be better to inherit from `HTTPError` directly?

Reply:
Yeah, I agree. I was just worried that the exception might be caught somewhere and if they only used `AmidError` it wouldn't catch this if it no longer inheritted from it. An alternative could be to change it to set the message attribute on itself and the parent passes it to super.

Reply:
Since the scope of this modif is only fix/revert previous regression, I approve it.
But feel free to improve it if you want (and yes I prefer your alternative to keep capacity to catch `AmidError`)